src/test/java/org/elasticsearch/rest/action/admin/indices/upgrade/UpgradeTest.java

```diff
@@ -246,6 +243,7 @@ public static void runUpgrade(HttpRequestBuilder httpClient, String index, Strin
         assertEquals(200, rsp.getStatusCode());
     }
 
+    @SuppressWarnings("unchecked")
```

Comment:
I don't have a problem with the change.  I was trying to understand why it was here.